src/components/tx/TxAlerts.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState, useEffect } from 'react';
+
+
+const TxAlerts = (props) => {
```

Comment:
Why don't we use classes for this component?

Reply:
After the introduction of hooks, functional components has become best practice for React. Even the docs encourage it: https://reactjs.org/docs/hooks-faq.html#should-i-use-hooks-classes-or-a-mix-of-both